### PR TITLE
feat(profile): re-design Contacts setting view

### LINKED ISSUE #5004: Implement new designs for the Contacts settings in the profile
Design: https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=875%3A116265

Needs this PR first: https://github.com/status-im/status-desktop/pull/5003

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #5004

Branched on top of https://github.com/status-im/status-desktop/pull/5003

StatusQ PR: https://github.com/status-im/StatusQ/pull/580

image

### CODE REVIEW

File: ui/app/AppLayouts/Profile/views/ContactsView.qml [hunk 1]
```diff
@@ -65,115 +67,145 @@ Item {
...
-                }
+            StatusTabButton {
+                id: pendingRequestsBtn
+                enabled: root.contactsStore.contactRequestsModel.count > 0
+                anchors.left: contactsBtn.right
+                anchors.top: parent.top
+                anchors.leftMargin: Style.current.bigPadding
+                btnText: qsTr("Pending Requests")
+            }
+            // StatusTabButton {
```
Comment (alaibe) [inline ui/app/AppLayouts/Profile/views/ContactsView.qml:95]: Shall this be remove or uncommented?

**Review by alaibe — APPROVED** [review]

File: ui/app/AppLayouts/Profile/views/ContactsView.qml (hunk 1 above)
Comment (jrainville) [reply ui/app/AppLayouts/Profile/views/ContactsView.qml:95]: Removed. I added it because we'll need it later, but it's not ready yet

**Comment by anastasiyaig** [comment]
@jrainville should the counter for Pending Requests be present in this PR or it is another task?  I mean i have 1 pending contact request, but there is no counter for Messaging section neither for pending requests tab 

https://user-images.githubusercontent.com/82375995/158951520-5e5ab82c-55d0-450b-96ea-1d0e69d01e16.mov

**Comment by anastasiyaig** [comment]
test run is here for this https://ethstatus.testrail.net/index.php?/runs/view/8759 , however the functionality will be enhanced therefore it will be updated as well

**Review by anastasiyaig — APPROVED** [review]

**Comment by anastasiyaig** [comment]
the suggestions above will be addressed in separate tickets

**Comment by jrainville** [comment]
New issues: 
- https://github.com/status-im/status-desktop/issues/5089
- https://github.com/status-im/status-desktop/issues/5090

**Pull request merged**